docs/safe-wallet-guide.md

```diff
@@ -3,383 +3,8 @@
 > [!WARNING]  
 > **Please do not edit this document in GitHub**!
 > 
-> The Safe Wallet Guide has been transitioned to a [Google Doc](https://docs.google.com/document/d/1jrxQ2ASDeSt3ZQqmRG4i1gGj5bLPIuNE/edit?usp=drive_link&ouid=111902059063316257481&rtpof=true&sd=true) and will be ready for general review and updates by **08-DEC**
+> The Safe Wallet Guide has been transitioned to a [Google Doc](https://github.com/openwallet-foundation/safe-wallet-sig/blob/main/docs/safe-wallet-guide.md) for collaborative editing.
```

Comment:
The copy suggests a Google document however all the updated links point to the `.md` file in GitHub. 

How the user will follow the instructions at the end of this document 
> Please see the Google Doc above for content.